tendrl/monitoring_integration/__init__.py

```diff
@@ -0,0 +1,75 @@
+import __builtin__
+import maps
+import traceback
+import sys
+import json
+from grafana.connection import get_conf
+from grafana.dashboard import create_dashboard
+from grafana.dashboard import set_home_dashboard
+from grafana.dashboard import get_dashboard
+from grafana.dashboard import get_all_dashboards
+from grafana.datasource import create_datasource
+from grafana.exceptions import ConnectionFailedException
+
+
+def main():
+
+    dashboards = []
+    config = get_conf("../../grafana.conf.yml")
+    setattr(__builtin__, "NS", maps.NamedDict())
+    setattr(NS, "conf", config)
+
+    dashboards = get_all_dashboards()
+    
+    title = []
+
+    for dashboard in dashboards:
+        title.append(dashboard["uri"].split('/')[1])
+
+    for dashboard in config.dashboards:
+        if dashboard in title:
+            sys.stdout.write('\n' + "Dashboard " + str(dashboard) +
+                             " already exists" + '\n')
+            continue
+        response_status = create_dashboard("grafana/dashboards", dashboard)
+        if response_status == 200:
+            sys.stdout.write('\n' + "Dashboard " + str(dashboard) +
+                             " uploaded successfully" + '\n')
+        else:
+            sys.stdout.write('\n' + "Dashboard " + str(dashboard) +
+                             " upload failed with error code " +
+                             str(response_status) + '\n')
+    try:
+        dashboard_json = get_dashboard(config.home_dashboard)
+
+        if 'dashboard' in dashboard_json:
+            dashboard_id = dashboard_json.get('dashboard').get('id')
+            response = set_home_dashboard(dashboard_id)
+            if response.status_code == 200:
+                sys.stdout.write('\n' + "Dashboard " +
+                                 str(config.home_dashboard) +
+                                 " is set as home dashboard" + '\n')
+        else:
+            sys.stdout.write('\n' + str(dashboard_json.get('message')) + '\n')
+    except ConnectionFailedException:
+        traceback.print_exc()
+        raise ConnectionFailedException
+
+    response_status = []
+
+    for datasource in config.datasource:
+        datasource_config = get_conf("../../" + str(datasource) + ".conf.yml")
```

Comment:
If this is loading graphite.conf.yml, please remove it and use /etc/monitoring-integration/monitoring-integration.conf.yml